src/languages/dotnet.md

```diff
@@ -0,0 +1,83 @@
+# .NET Core
+
+
+Platform.sh supports deploying .NET Core applications.
+
+## Supported versions
+
+* 2.0
+* 2.1
+* 2.2
+
+To specify a dotnet container, use the `type` property in your `.platform.app.yaml`.
+
+```yaml
+type: 'dotnet:2.2'
+```
+
+## Building the application
+
+Dotnet containers do not come with a pre-defined build process that you may have seen on other container types. Instead, the process is defined by you, using the runtime and output directory variables passed to the build environment.
```

Comment:
What is this supposed to mean? As far as I'm aware, only PHP has a defined build process with native composer support, but I don't think it warrants this paragraph. Unless this is just referred to the `dependencies` block, in which case, I'd say something like "does not have automatic dependency installation" or something to that effect.

Reply:
Only this container has a separate source and output folder.